acrobat/blocks/mobile-widget/mobile-widget.js

```diff
@@ -106,16 +118,143 @@ function createMobileWidget(element, content, verb) {
   });
 }
 
+window.addEventListener('milo:modal:closed', () => {
+  if (!isAcrobatExitOpen) return;
+  mobileAnalytics(aaVerbName, 'acrobat-exit-modal:closed');
+  isAcrobatExitOpen = false;
+  sessionStorage.setItem('modalDismissed', 'true');
+  if (externalLinkHref) {
+    window.location.href = externalLinkHref;
+    externalLinkHref = '';
+  }
+});
+
+const getPlaceHolders = async () => {
+  const { getConfig } = await import(`${miloLibs}/utils/utils.js`);
+  const config = await getConfig();
+
+  if (!Object.keys(window.mph || {}).length) {
+    const placeholdersPath = `${config.locale.contentRoot}/placeholders.json`;
+    try {
+      const response = await fetch(placeholdersPath);
+      if (response.ok) {
+        const placeholderData = await response.json();
+        placeholderData.data.forEach(({ key, value }) => {
+          window.mph[key] = value.replace(/\u00A0/g, ' ');
+        });
+      }
+    } catch (error) {
+      window.lana?.log(`Failed to load placeholders: ${error?.message}`);
+    }
+  }
+};
+
+async function showModal(event = 'shown') {
+  await getPlaceHolders();
+  const { getModal } = await import(`${miloLibs}/blocks/modal/modal.js`);
+  const acrobatIcon = createTag('img', { class: 'modal-icon', src: '/acrobat/img/icons/widget-icon.png' });
+  const acrobatLabel = createTag('p', { class: 'modal-label' }, 'Adobe Acrobat');
+  const acrobatLogo = createTag('div', { class: 'modal-logo' });
+  acrobatLogo.append(acrobatIcon, acrobatLabel);
+  const modalTitle = createTag('h2', { class: 'modal-title' }, (window.mph && window.mph['acrobat-exit-modal-title']) || 'Before you go, get the free mobile app.');
+  const modalMessage = createTag('p', { class: 'modal-message' }, (window.mph && window.mph['acrobat-exit-modal-message']) || 'Quickly comment, sign, and share PDFs — all from your phone.');
+  const modalCta = createTag('a', { class: 'modal-cta', href: (window.mph && window.mph['acrobat-exit-modal-cta-url']) || '/acrobat/mobile-app.html' }, (window.mph && window.mph['acrobat-exit-modal-cta-label']) || 'Download app');
+  modalCtaLink = modalCta.href;
```

Comment:
Repeatedly checking `window.mph`
```
(window.mph && window.mph['acrobat-exit-modal-cta-label']) || 'Download app'
```
How about using another variable and simplify them?
```
const mph = window.mph || {};

mph['acrobat-exit-modal-cta-label'] || 'Download app'
```